src/components/Article.jsx

```diff
@@ -1 +1,51 @@
-// export const Article = () => ();
+import PropTypes from 'prop-types';
+
+import React from 'react';
+
+export const Article = ({
+  title,
+  date,
+  text,
+}) => {
+  const today = new Date(date);
+  const dateISO = today.toISOString().slice(0, 10);
+  const dateLocaleString = today.toLocaleDateString('en-US', {
```

Comment:
`date` is already what you need, you just create a copy with `new Date()`, but there is no actual purpose 
```suggestion
  const dateISO = date.toISOString().slice(0, 10);
  const dateLocaleString = date.toLocaleDateString('en-US', {
```